docs/src/reference-manual/cbmc-starter-kit-setup-ci.md

```diff
@@ -0,0 +1,58 @@
+# Reference manual
+
+## Name
+
+`cbmc-starter-kit-setup-ci` - Add GitHub Actions workflow to run proofs as part of CI
+
+## Synopsis
+
+```
+cbmc-starter-kit-setup-ci [-h] --github-actions-runner
+                                 <ubuntu-20.04>|<name-of-your-Ubuntu-20.04-large-runner>
+                                 [--cbmc <latest>|<X.Y.Z>]
+                                 [--cbmc-viewer <latest>|<X.Y>]
+                                 [--litani <latest>|<X.Y.Z>]
+                                 [--kissat <latest>|<TAG>]
+                                 [--cadical <latest>|<TAG>] [--verbose]
+                                 [--debug] [--version]
+```
+
+## Description
+
+This script adds a GitHub Actions workflow to your GitHub repository, such that
```

Comment:
Suggest breaking this sentence up, the information about runners doesn't need to read immediately.

> This script adds a GitHub Actions workflow that runs all CBMC proofs as part of your project's CI phase. The proofs run on Ubuntu 20.04 on a standard GitHub-hosted Runner by default, but can also be configured to use a [Large Runner](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners) that you have created.

Reply:
I will break this up and reuse some of the text present in the `## Running CBMC proofs as part of CI` heading of the installation `README`.